aspnetcore/security/authentication/identity-api-authorization.md

```diff
@@ -101,14 +102,14 @@ After the call to `builder.Build()`, call <xref:Microsoft.AspNetCore.Routing.Ide
 app.MapIdentityApi<IdentityUser>();
 ```
 
-## Secure endpoints
+## Secure selected endpoints
```

Comment:
Should we mention how to secure with a specific claim or permission too? I.e. the difference between:

`RequiresAuthorization()`  

and

`RequiresAuthorization("Admin")`